amundsen_application/static/js/components/Feedback/FeedbackForm/RatingFeedbackForm/index.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+import AbstractFeedbackForm, { DispatchFromProps, StateFromProps } from '../../FeedbackForm';
+
+import { GlobalState } from "../../../../ducks/rootReducer";
+import { submitFeedback, resetFeedback } from '../../../../ducks/feedback/reducer';
+
+export class RatingFeedbackForm extends AbstractFeedbackForm {
+  constructor(props) {
```

Comment:
Do we need to call the constructor method of each subclass of `AbstractFeedbackForm` ? If not, we can remove them also, as they are not serving any specific purpose here.

Reply:
👍 Feel free to remove in a PR.